### ISSUE TITLE
Can't find developer community or the documentation guild

### ISSUE THREAD

**Issue by MalcolmAnderson** [aidancbrady/Mekanism#4936]
#### Issue description:

I can find the issues list (here), and I can find the mod's forum post (listed in the README.md) but I can't find the google list, or slack space, or some other place where the devs hang out.

Where this is coming from is that I'd like to add things to the docs like build instructions and contributor guidelines (things I wish that I had before I got started).

Is there a community where I run things by people, or do I just post an issue like this?

If there is some kind of community hang out where do I find it, 
and where in the docs did I miss the mention of it?

**Comment by thiakil** [comment]
The best place is indeed here at the moment.

**Comment by MalcolmAnderson** [comment]
Ok then.
I just added a PR (Add build instructions and contributing guidelines #4937)
With the comments:
Related to my question about documentation
https://github.com/aidancbrady/Mekanism/issues/4936
This PR is as much the start of a conversation as it is a PR.

The main intent is to add documentation to make it easier for new people to get in and get started in either using the latest bits, and / or starting on their open source journey.

The change to the README.md file is:
A single line pointing an interested person to the docs directory.

The build instructions file are:
My instructions are just what I figured out on my own.  They may or may not be the right instructions, but they "worked on my machine"
Eventually this file should expand to subjects like "configuring mekanism for your server" or anything else that people think is appropriate.

The contributing guidelines are:
intended to point people in the right direction, but I suspect that it will change wildly over time
Eventually this file should expand to subjects like, "setting up your build environment", "issue reproduction tips and tricks", "reading and understanding the developer logs"
